cmd/skaffold/app/cmd/debug.go

```diff
@@ -44,7 +44,7 @@ func NewCmdDebug() *cobra.Command {
 
 func runDebug(ctx context.Context, out io.Writer) error {
 	opts.PortForward.ForwardPods = true
-	deploy.AddManifestTransform(debugging.ApplyDebuggingTransforms)
+	manifest.AddManifestTransform(debugging.ApplyDebuggingTransforms)
```

Comment:
should we rename `AddManifestTransform` to `AddTransform` since it's moved into the `manifest` package?

Reply:
sure good call